docs/apache-airflow/logging-monitoring/errors.rst

```diff
@@ -29,13 +29,14 @@ First you must install sentry requirement:
 
    pip install 'apache-airflow[sentry]'
 
-After that, you need to enable the integration by set ``sentry_on`` option in ``[sentry]`` section to ``"True"``.
+After that, you need to enable the integration by set ``sentry_on`` option in ``[sentry]`` section to ``'True'``.
```

Comment:
```suggestion
After that, you need to enable the integration by set ``sentry_on`` option in ``[sentry]`` section to ``True``.
```

Reply:
Should I remove the single quotes for `sentry_dsn` as well?

Reply:
+1 to removing quotes to both. These are INI, not Python, the quotes just shouldn’t exist in the first place.

Reply:
yes please remove the quotes